pages/wiki/watchfaces-creation.md

```diff
@@ -77,6 +78,30 @@ layout: documentation
     }</code></pre>
   <p>The UI slider then acts as a controller to see how the watchface reacts to the different values. Note that the slider gives a real value from 0.0 to 1.0. To simulate the integer 0 to 100 provided by the real battery on the watch, we scale and convert to a fixed value in the code above.</p>  
 </div>
+<div class="page-header">
+  <h1 id="how-to-share">Share your watchface</h1>
+</div>
+<div>
+  <p>Pull requesting your finished watchface to the <a href="https://github.com/AsteroidOS/unofficial-watchfaces">unofficial-watchfaces repo</a> is always very welcome! Follow this guide to ensure consitency with existing community watchfaces and a quick review process.</p>
```

Comment:
I find that the first sentence feels a bit weird (the 'Pull requesting' part). What do you think about this:
Your finished watchface is always welcome on our unofficial-watchfaces repository! You can do this via pull requests.

In my opinion it focuses on You and your watchface instead of pull requesting (which is just the method to get your work in our repository).

Reply:
Very good finds!
Agreed on all, thanks for the improvement.